### ISSUE TITLE
spatie/ignition default config outside of appdir open_basedir issue

### ISSUE THREAD

**Issue by dianfishekqi** [spatie/ignition#22]
Hello there, 

This [issue](https://github.com/spatie/laravel-ignition/issues/19)  seems to be caused by spatie/ignition package.

Steps to reproduce:
1. Have a Laravel 8.xx app and open_basedir  restrictions set to the application directory
2. replace `facade/ignition` with `spatie/laravel-ignition`

I can confirm that homedir being the default path causes the issue after changing the default path to `base_path` everything works normally.

This can cause problems with the upcoming Laravel 9 release, as there it is the default.

cheers

**Comment by dianfishekqi** [comment]
Hello there, can you check this out, still can't use the package because of default config sits at `~` outside of the laravel basepath?

@rubenvanassche  @freekmurze

**Comment by fightborn** [comment]
20 days and no reply? Like, nothing? 

"Beautiful" error page that one can't get rid of and if something goes wrong - the support is no where to be found. Awesome. And this is core part of most popular framework...

**Comment by freekmurze** [comment]
@fightborn Please keep it friendly. If this is urgent for you doesn't necessarily make it urgent for us.

Here are a couple of options:
1. Try to PR a fix.
2. Remove ignition from composer. Laravel will still work, and you'll see a more basic error page.

Good luck!

**Comment by dianfishekqi** [comment]
Hello @freekmurze, I'm willing to submit a PR, what would be acceptable behavior for you?

Continuing to use the home dir as the default path and allowing to change the path in a config file or changing the default path to `base_dir`.

Thank you!

**Comment by freekmurze** [comment]
Thank you! Allowing the path to be changed in the config file seems like the route to go for me.

**Comment by fightborn** [comment]
@freekmurze you made it urgent for me by hijacking Laravel and turning into adware. If anyone should keep it polite, it's you. Fucking incompetent bunch of morons, from introducing CVE to not being able to check your damn code before trying to sell it as gospel.

Fuck you,
sincerely - someone you made use your crappy products.

**Comment by leafling** [comment]
I came across this issue when I upgraded to Laravel 9.  Rather than override this with a config file setting, would it make sense to compare the desired .iginition.json path against the open_basedir paths to see if the users home directory is allowed?  Then it could be seamless and there would be no need to worry about a separate config file setting and no need to direct people to manually add a config setting to override what is likely to be a pretty common problem -- for pretty much everyone that uses open_basedir restrictions.
